Wikipedia/Code/TabsOverviewCoordinator.swift

```diff
@@ -148,20 +150,20 @@ final class TabsOverviewCoordinator: Coordinator {
                 displayDeleteAllTabsToast: displayDeleteAllTabsToast
             )
             
-            let articleTabsView = WMFArticleTabsView(viewModel: articleTabsViewModel)
-            let hostingController = WMFArticleTabsHostingController(
+            let articleTabsView = await WMFArticleTabsView(viewModel: articleTabsViewModel)
```

Comment:
Maybe in the future we should `@MainActor` around TabsOverviewCoordinator instead. Not a blocker though.